jest.conf/visual.testUtils.js

```diff
@@ -0,0 +1,43 @@
+import percySnapshot from '@percy/puppeteer';
+
+export async function renderComponent(component, props) {
+  const initialState = await page.evaluate(() => {
```

Comment:
I think we could improve this name to something like `beforeRenderState` or `beforeRenderSnapshot` so the name suggests we are going to do something with this variable after rendering the component

Reply:
I agree with that. Will make the changes soon :+1: